src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -44,27 +63,53 @@ public static class GenAPIApp
 
             string headerFileText = ReadHeaderFile(headerFile);
 
-            AccessibilitySymbolFilter accessibilitySymbolFilter = new(
+            ISymbolFilter typeFilter = new AccessibilitySymbolFilter(
```

Comment:
Why did you change this?

Reply:
Because I will redefine it later as a different type of filter.